vultr/resource_vultr_container_registry_test.go

```diff
@@ -0,0 +1,60 @@
+package vultr
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccVultrContainerRegistry(t *testing.T) {
+	crName := acctest.RandomWithPrefix("tf-cr-cr")
```

Comment:
```
    resource_vultr_container_registry_test.go:17: Step 1/1 error: Error running apply: exit status 1

        Error: error creating container registry: {"error":"Registry Name can only be lowercase alpha numeric characters","status":422}

          with vultr_container_registry.foo,
          on terraform_plugin_test.tf line 2, in resource "vultr_container_registry" "foo":
           2:           resource "vultr_container_registry" "foo" {
```

Reply:
Probably should add some validation for this in the provider, but that can happen later

Reply:
I followed the Vultr API documentation, perhaps I forgot to validate the field. Thanks for your feedback.
https://www.vultr.com/api/#tag/Container-Registry/operation/create-registry

Reply:
Maybe we can transform the name to lowercase before sending the create request.